### ISSUE TITLE
[New Core]: JTCOMSC

### ISSUE THREAD

**Issue by neil-morrison44** [openfpga-cores-inventory/analogue-pocket#41]
### Core Author username

jotego

### Core .yml snippet

```yml
display_name: JTCOMSC
repository: jtbin
path: pocket/zips/jotego.jtcomsc.zip
```

**Comment by neil-morrison44** [comment]
@joshcampbell191 Might be able to use a GitHub app to get around the rate limiting issue - I think they’ve got really high limits (50,000 requests an hour or something)

**Comment by joshcampbell191** [comment]
Addressed via #46
